mitodl/ocw-hugo-themes
# Add shortcode for resource links

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #173 

#### What's this PR do?
Adds a resource shortcode which renders as a simple link (cc @alicewriteswrongs)

#### How should this be manually tested?
 - Convert `11-360-community-growth-and-land-use-planning-fall-2010` using ocw-to-hugo to markdown
 - Edit `content/pages/assignments.md` to add this shortcode right below the "Background & Project Area" header: `{{< resource_link "8bdc0e15fab4a6d317dbe0b3e8a58729" "a link to projects" >}}`
 - Edit `.env` to reference the course and run `npm run start:course`. Go to http://localhost:3000/pages/assignments/. You should see a link near the top that says "a link to projects". 
 - Click the link and you should be directed to `/pages/projects/`
